### PR TITLE
iOS setup instructions for writing code on device

### PR DESCRIPTION
**Pull request by TheMaja**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [ ] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
1. Looking at the android setup, it looks like you are missing a few things:
- [x] code editor
- [ ] Topic: Python setup for IOS
- [ ] Topic: Javascript setup for IOS
- [ ] Topic: Java setup for IOS

Having at least one of those language setup shows that code will actually be successfully written with IOS. If not even one of the language setup can be done successfully then I don't know if it's worth saying we have an IOS setup at the moment.

2. Please also convert some of the links in this topic to use markdown

File: content/environment-setup/git/_index.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Git is a critical tool for any kind of coder. We'll get git set up before you do
 
 ## If you have an Android device
 
-Start by installing Termux: https://play.google.com/store/apps/details?id=com.termux&hl=en_ZA
+Start by installing Termux: [https://play.google.com/store/apps/details?id=com.termux&hl=en_ZA](https://play.google.com/store/apps/details?id=com.termux&hl=en_ZA)
```
Comment (Ngoakor12) [inline content/environment-setup/git/_index.md]: Including the link in the square and round brackets defeats the whole purpose of markdown links and semantic html once everything is live. The text in the square brackets should be something descriptive that describes the link just like you did with the other links

File: content/environment-setup/git/_index.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ Git is a critical tool for any kind of coder. We'll get git set up before you do
...
 
 Note: Since November 2022, the Google Playstore has refused to accept updates from Termux, so downloading Termux from there won't work. You can download the latest version of Termux from [https://f-droid.org/en/packages/com.termux/](https://f-droid.org/en/packages/com.termux/).
```
Comment (Ngoakor12) [inline content/environment-setup/git/_index.md]: Same feedback regarding links here too

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
Nice! Almost there... 

Unfortunately I can't review the IOS setup in depth.

Can we change "iphone" to "IOS" everywhere to be inclusive of all devices that use IOS just like we have with Android. Make sure the change does not affect readability

File: content/environment-setup/git/_index.md [hunk 3]
```diff
@@ -44,7 +43,7 @@ cd storage/shared
 # now you can cd into your folders and interact with git
 ```
 
-This video walks you through the process:  https://www.youtube.com/watch?v=DG3l9sxFVnY
+This [video](https://www.youtube.com/watch?v=DG3l9sxFVnY) walks you through the process.
 
 Note:
```
Comment (Ngoakor12) [inline content/environment-setup/git/_index.md:49]: you missed converting this link: https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/ to be proper markdown

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
Almost there...

**Review by Ngoakor12 — APPROVED** [review]

File: content/environment-setup/javascript-on-ios/_index.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
+---
+content_type: topic
+ready: true
+title: javaScript setup for iOS
```
Comment (sheenarbw) [inline content/environment-setup/javascript-on-ios/_index.md:4]: javaScript should have a capital J

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**